Sources/ArcGISToolkit/Components/Augmented Reality/FlyoverSceneView.swift

```diff
@@ -187,3 +187,25 @@ private class ObservableARSession: NSObject, ObservableObject, ARSessionDelegate
         currentFrame = frame
     }
 }
+
+extension FlyoverSceneView {
+    func getInitialCamera() -> Camera {
+        return initialCamera
+    }
+    
+    func getTranslationFactor() -> Double {
+        return translationFactor
+    }
+    
+    func getCameraController() -> CameraController {
+        return cameraController
+    }
+    
+    func getShouldOrientToCompass() -> Bool {
+        return shouldOrientToCompass
+    }
+    
+    func getInterfaceOrientation() -> InterfaceOrientation? {
+        return interfaceOrientation
+    }
+}
```

Comment:
I can understand doing this for the camera controller and interface orientation because state properties are supposed to be private by convention but for the others, any reason to not just make the properties themselves internal?

Reply:
If we need to test these, then the properties themselves should just be internal.

Reply:
> I can understand doing this for the camera controller and interface orientation because state properties are supposed to be private by convention but for the others, any reason to not just make the properties themselves internal?

No, I will make those properties internal.